uploader.json

```diff
@@ -1287,6 +1287,11 @@
             "pattern": [],
             "symbolic_name": "cciss"
         },
+        {
+		    "file": "/sys/bus/usb/drivers/cdc_wdm/module/refcnt",
+		    "pattern": [],
```

Comment:
@vishwanathjadhav please fix the spacing here.

Reply:
@bfahr - Done